ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.11-presubmits.yaml

```diff
@@ -162,10 +162,9 @@ presubmits:
           value: e2e-aws
         - name: RPM_REPO_OPENSHIFT_ORIGIN
           value: https://rpms.svc.ci.openshift.org/openshift-origin-v3.11/
-        - name: TEST_FOCUS
-          value: Suite:openshift/conformance/parallel/minimal
-        - name: TEST_SKIP
-          value: (\\[local\\]|\\[templates\\]|\\[Feature:Router\\]|Kubectl logs)
+        - name: TEST_COMMAND
+          value: TEST_FOCUS=Suite:openshift/conformance/parallel/minimal TEST_SKIP='(\\[local\\]|\\[templates\\]|\\[Feature:Router\\]|Kubectl
```

Comment:
1. What format does `ginkgo` expect?
2. There is no semantic change here.
3. If it worked before, it will still work. If it didn't, it won't.
4. /shrug?

Reply:
yeah agreed on all counts

Reply:
I think `\\[` is correct -- we want a literal `\[` going to the Ginkgo regex, so whoever is using `\[` is wrong.

Reply:
(out of band from this PR)